_posts/2021-03-17-prettifying-long-commit-histories.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Prettifying Long Commit Histories with Git"
+date: 2021-03-17 11:34:00
+tags:
+- Documentation
+- GitHub
+- Git
+categories:
+- Documentation
+- GitHub
+- Git
+twitter_text: Prettifying Commit Histories 
+authors: Steliana Vassileva
+image: /images/documentation/git-commit-history.png
+image_url: https://tugberkugurlu.blob.core.windows.net/bloggyimages/d773c1fe-4db8-4d2f-a994-c60f3f8cb6f0.png
+---
+
+We often use GitHub to create a compare view, and list pull requests (PRs) for releases. However, GitHub limits the number of commits you can view at a time. If the difference between hashes is greater than 250 commits, you're delivered a disappointing message.
+
+![GitHub Most Recent 250 Commits](/images/documentation/github-compare-most-recent-250-commits.png)
+
+Since I write release notes, I need to view the entire list of pull requests. So this scenario makes me frown a bit. But recently, a helpful **git log** option brightened up my day.
+
+## The git log basics
+
+The **git log** command [shows commit logs](https://git-scm.com/docs/git-log). Without any arguments, we use it to view commit history, with the most recent commits displaying first. SHA-1 checksum, author, date, and commit message are usually included.
+
+However, numerous options can make the **git log** command more robust. These helpers can narrow down, embellish, or prettify results.
+
+Here we focus on the **oneline** option, which serves as a shorthand for:
+
+`--pretty=online --abbrev-commit`
+
+## Time for some Git magic
+
+With the steps below, you can generate a complete list of PRs in your CLI. The assumption is that the upstream **master** branch is the source of truth for your project and the development branch.
```

Comment:
first use of the "CLI" acronym? or the "PR" acronym?

Reply:
Good point on CLI. Made an assumption. Will update. The first sentence in the post addresses the PR acronym.